lib/client.ex

```diff
@@ -49,6 +52,7 @@ IO.puts "running via tasks"
 IO.puts "running via serial map"
 {timing_serial, _result} = :timer.tc(SerialRunner, :execute, [connections, &Client.get_JSON/0])
 
+IO.puts "running a sinle request took #{single_request_timing} microseconds"
```

Comment:
Typo on single; did a rebase locally, but not sure why it is not taking affect on this pull request.